src/Illuminate/Contracts/Support/ValidatedData.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Illuminate\Contracts\Support;
+
+use ArrayAccess;
+use IteratorAggregate;
+
+interface ValidatedData extends Arrayable, ArrayAccess, IteratorAggregate
+{
+    //
+}
```

Comment:
Do we want the interface to actually show any methods that need to be supported, e.g. `only()`, `except()`, ...etc.